pyproject.toml

```diff
@@ -43,7 +43,7 @@ classifiers = [
 ]
 
 [project.optional-dependencies]
-doc = [
+doc_base = [
```

Comment:
one comment: the failing tests are because only half of the packages are installed as long as there is no version released with the changes:
```
WARNING: pydata-sphinx-theme 0.12.1rc1.dev0 does not provide the extra 'doc-base'
```